_posts/2023-05-17-save-date-ugm2023.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Save the Date - MDAnalysis UGM 2023
+---
+
+We are thrilled to announce the first ever MDAnalysis User Group Meeting (UGM)! The MDAnalysis UGM 2023 will take place September 27-29 (Wednesday-Friday) at the [University of Lisbon][uniL] (Portugal). To foster a connection opportunity for interdisciplinary researchers and developers across biomolecular simulations, soft matter and biophysics, this year's event will be held in-person and free to attend for all.
```

Comment:
Do we have to say here that it's free or can we provide details in a later announcement with registration details? That would give us more flexibility.

How are we handling registration? Will there be a wait list? How are we preventing no-shows from taking up seats?